skimage/filters/thresholding.py

```diff
@@ -801,8 +801,10 @@ def _mean_std(image, w):
     ----------
     image : ndarray
         Input image.
-    w : int
-        Odd window size (e.g. 3, 5, 7, ..., 21, ...).
+    w : int, tuple
```

Comment:
at the beginning of the docstring you should also modify "rectangular window with size w times w".